docs/quiz-toolkit/google-sheet.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: "Google Sheets"
+sidebar_position: 2
+---
+# Google Sheets Integration
+Users can append data to their own google sheets by calling [`append_to_google_sheet`](../mecsimcalc-library#append_to_google_sheet). By linking MecSimCalc directly to a Google Sheet, all responses from quizzes can be instantly recorded and stored in a structured format, ready for further use.
+
+In this part, we will teach you how to find the `spreadsheet_id` and `service_account_info` parameters for `append_to_google_sheet` function
```

Comment:
Instead of this. Explain that MecSimCalc has its own function that can be used. Show the function and all the arguments.  (Similar to the library page)
Show an example code (from a template app) that uses this functions and sends the student name, email, id, score, timestamp